curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dfa30b9eacea3f48c6300ad.md

```diff
@@ -29,7 +29,7 @@ Your anchor (`a`) element should have an opening tag. Opening tags have this syn
 assert(document.querySelectorAll('a').length >= 2);
 ```
 
-You should only add one opening anchor (`a`) tag. Please remove any extras.
+You are missing a closing (`a`) tag after the image.
```

Comment:
I don't think this following test is actually checking for a second closing tag. There are many ways to not have exactly 2 complete `a` elements.